src/google/protobuf/compiler/parser.cc

```diff
@@ -1373,7 +1373,7 @@ bool Parser::ParseOption(Message* options,
           value_location.AddPath(
               UninterpretedOption::kNegativeIntValueFieldNumber);
           uninterpreted_option->set_negative_int_value(
-              -static_cast<int64>(value));
+              static_cast<int64>(-value));
```

Comment:
Let's keep "-value" for now. We can change latter if it turns out to be problematic.